src/components/DetailView/vegaSpec.js

```diff
@@ -137,8 +144,8 @@ export function createSpec(sensor, primaryValue, selections, initialSelection, t
       contains: 'padding',
       resize: true,
     },
-    padding: { left: 50, right: 2, top: 45, bottom: 5 },
-    transform: sensor.hasStdErr ? stdErrTransform : [],
+    padding: { left: leftPadding, right: 2, top: 50, bottom: 5 },
+    transform: [...(sensor.hasStdErr ? stdErrTransform : [])],
```

Comment:
This used to have the array wrapper around spreading of the elements, but it got dropped at some point.  However, now that I think about it, they are equivalent, so I'll put it back to the way it was.